src/ggplotnim/ggplot_vega.nim

```diff
@@ -292,3 +343,58 @@ proc `+`*(p: GgPlot, d: VegaDraw) =
   let outname = writeVegaFile(plt, d)
   if d.show:
     showVega(outname, d)
+
+proc toVegaHtml*(
+  p: GgPlot,
+  width = 640.0, height = 480.0,
+  pretty = false,
+  divName = "div",
+  onlyBody = false,
+  vegaCDN = "https://cdn.jsdelivr.net/npm/",
```

Comment:
maybe `vegaLibsPath` instead of `vegaCDN`? it is there so that you can use a static copy instead of CDN right?

Reply:
Well, I added it mainly so that it's changeable without a PR if the URL suddenly changes, but of course it can also be used for a static copy indeed! I'll change the name. :ok_hand:

Reply:
ah right, also changes of cdn are something to account for! :)